backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -95,41 +103,37 @@ public TestResultUpload processResultCSV(InputStream csvStream) {
     List<FeedbackMessage> errors =
         testResultFileValidator.validate(new ByteArrayInputStream(content));
     if (!errors.isEmpty()) {
-      result.setErrors(errors.toArray(FeedbackMessage[]::new));
-      return result;
+      validationErrorResult.setErrors(errors.toArray(FeedbackMessage[]::new));
+      return validationErrorResult;
     }
 
     if (!"P".equals(processingModeCodeValue)) {
       content = attachProcessingModeCode(content);
     }
 
-    UploadResponse response = null;
+    TestResultUpload csvResult = null;
+    Future<UploadResponse> csvResponse;
+    Future<UploadResponse> fhirResponse = null;
     if (content.length > 0) {
-      try {
-        response = _client.uploadCSV(content);
-      } catch (FeignException e) {
-        log.warn("Bulk test result upload unsuccessful", e);
-        response = parseFeignException(e);
+      csvResponse = submitResultsAsCsv(content);
+      if (fhirEnabled) {
+        fhirResponse = submitResultsAsFhir(new ByteArrayInputStream(content), org);
```

Comment:
Correct - same situation as the single entry flow.  Existing pipeline sends covid and drops any multiplex/flu results on the ground.  New pipeline will filter out covid to prevent duplication.